index.html

```diff
@@ -289,10 +304,11 @@ <h3>Normative Specifications</h3>
                             <b>Expected completion:</b> WG-START + 24 months.
                         </p>
                     </dd>
+
                     <dt id="vocabulary" class="spec">Linked Data Security Vocabulary (LDSV)</dt>
                     <dd>
                         <p>
-                            This specification defines an RDF Vocabulary to express the results of the Linked Data Security processes and their parameters in an RDF Dataset in a way that enables a 3rd party to verify the result. The specification also defines a preferred JSON-LD Context document to be used by the JSON-LD serialization.
+                            This specification defines a formal RDF Vocabulary for the terms defined in the Linked Data Integrity deliverable. The specification also defines a preferred <a href="https://www.w3.org/TR/json-ld11/#the-context">JSON-LD Context</a> document to be used by a JSON-LD serialization.
```

Comment:
It may turn out that the context defined may not actually be preferred -- but just one example that could be used. It may also be that a number of contexts are defined or a process is recommended for defining them with narrowly scoped terms. Trying to give flexibility here.

```suggestion
                            This specification defines a formal RDF Vocabulary for the terms defined in the Linked Data Integrity deliverable. The specification may also define one or more <a href="https://www.w3.org/TR/json-ld11/#the-context">JSON-LD Context</a> documents to be used by a JSON-LD serialization.
```